examples/python/easy_start_demo/client.py

```diff
@@ -0,0 +1,98 @@
+"""
+Author: Zhongying Wang
+Email: kerbalwzy@gmail.com
+DateTime: 2019-08-13T23:30:00Z
+PythonVersion: Python3.6.3
+"""
+import os
+import sys
+import time
+import grpc
+
+# add the `demo_grpc_dps` dir into python package search paths
+BaseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.join(BaseDir, "demo_grpc_pbs"))
```

Comment:
I don't think we want to be encouraging this type of path manipulation. Please use a more standard import mechanism.

Reply:
If I don't, I need to modify the automatically generated code, which in `demo_pb2.py` and `demo_bp2_grpc.py` created by grpc tools,  to ensure that the program works properly.

Reply:
I think the way, modify the code automatically generated by python grpc tool, is more unfriendly to others.

Reply:
Agree. You shouldn't have to modify the generated code *or* manipulate the path. Check out the [`hello_world` example](https://github.com/grpc/grpc/blob/master/examples/python/helloworld/greeter_server.py) or [this key-value store example](https://github.com/gnossen/grpc-example/blob/master/grpc_kv_server/src/grpc_kv_server/__init__.py#L12). I would actually discourage users from checking generated code into source control at all.